packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -1249,3 +1259,17 @@ export const PSTN_STATUS = {
   SUCCESS: 'SUCCESS', // happens after the transfer (TRANSFERRING) is successful
   UNKNOWN: '' // placeholder if we haven't been told what the status is
 };
+
+export const PASSWORD_STATUS = {
+  NOT_REQUIRED: 'NOT_REQUIRED', // password is not required to join the meeting
+  REQUIRED: 'REQUIRED', // client needs to provide the password by calling setPassword() before calling join()
+  UNKNOWN: 'UNKNOWN', // we are waiting for information from the backend if password is required or not
```

Comment:
yes, existing sdk code calls fetchMeetingInfo() immediately after the meeting object is created, but it is an async operation so someone might try to read the password status before fetchMeetingInfo() is completed. There are also other possibilities:
1. fetchMeetingInfo() may fail for some network/timeout reason
2. fetchMeetingInfo() may require captcha just because it was too many attempts to fetch info for non-existing meeting (code 423001)
in these 2 cases we will not know if password is required or not until we successfully try again